paket.dependencies

```diff
@@ -95,7 +95,7 @@ group Build
     nuget SourceLink.Fake
     nuget ILRepack
     nuget RoslynTools.ReferenceAssemblies
-
+    nuget Newtonsoft.Json
```

Comment:
Where does this come from?

Reply:
I referenced the #load DotNet module directly so i can use DotNet.Release_2_1_300_RC1. If i dont add the Newtonsoft.Json the Bootstrap test fails. I can remove it and just add the new version directly and update that later.